ChangeLog

```diff
@@ -1,3 +1,10 @@
+2015-05-02  Jim Shaver <dcypherd@gmail.com>
+
+	* .travis.yml, setup.py, tox.ini: Removed support for Python 3.2.
+	  This version is rarely used and is now deprecated by a major
+	  dependency of pyOpenSSL(cryptography). Affected users should upgrade
```

Comment:
I know this is complete bikeshedding but please add a space before `(cryptography)` and a second space before `Affected`.  Thanks!